Philippines/salamat.md

```diff
@@ -1,3 +1,5 @@
 Maligayang pasko at manigong bagong taon sa inyo! Salamat sa Udacity para sa
 
-online class na ito :D
\ No newline at end of file
+online class na ito :D
+
+Dahil sa bagong skills na ito, sa pag gamit ng Git at GitHub, mas mapapabilis ang pag gawa ng mga software at apps. Magiging organized pa para makita agad ang mga bugs. Pwede rin mag experimento sa pamamagitan ng pag gawa ng branch.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.